pkg/cluster/cluster.go

```diff
@@ -49,7 +49,7 @@ var (
 type Config struct {
 	OpConfig                     config.Config
 	RestConfig                   *rest.Config
-	PgTeamMap                    pgteams.PostgresTeamMap
+	PgTeamMap                    *pgteams.PostgresTeamMap
```

Comment:
did we not say that we could just not use this var at all and always just use

cluster: c.opconfig.pgteams?